src/views/v2/day/type-separator.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * View: Day View Type separator
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/day/type-separator.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ */
+
+use Tribe\Events\Views\V2\Utils;
+use Tribe__Date_Utils as Dates;
+
+$should_have_type_separator = Utils\Separators::should_have_type( $this->get( 'events' ), $event );
+
+if ( ! $should_have_type_separator ) {
+	return;
+}
+$separator_text = Utils\Separators::get_type_text( $event );
```

Comment:
I feel like we should have the logic on the template in this case.

Reply:
I was trying to avoid just that.